internal/cmd/local/command_kafka_start.go

```diff
@@ -26,21 +27,25 @@ import (
 	"github.com/confluentinc/cli/internal/pkg/output"
 )
 
-var (
-	statusBlacklist = []string{"Pulling fs layer", "Waiting", "Downloading", "Download complete", "Verifying Checksum", "Extracting", "Pull complete"}
-)
-
-type imagePullOut struct {
-	Status string `json:"status"`
+type ImagePullResponse struct {
+	Status   string `json:"status"`
+	Error    string `json:"error,omitempty"`
+	Progress string `json:"progress,omitempty"`
+	ID       string `json:"id,omitempty"`
 }
 
 func (c *Command) newKafkaStartCommand() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "start",
 		Short: "Start a single-node instance of Apache Kafka.",
 		Args:  cobra.NoArgs,
 		RunE:  c.kafkaStart,
 	}
+
+	cmd.Flags().String("kafka-rest-port", "8082", "The port number for Kafka REST.")
```

Comment:
Let's put this string in a constant and also use it in the localports struct?

Reply:
do you mean const `"kafka-rest-port"`? how do we use it in the struct?